src/marks/delaunay.js

```diff
@@ -279,9 +279,13 @@ function voronoiof(delaunay, dimensions) {
   return delaunay.voronoi([marginLeft, marginTop, width - marginRight, height - marginBottom]);
 }
 
-function delaunayMark(DelaunayMark, data, {x, y, ...options} = {}) {
+function delaunayMark(DelaunayMark, data, {x, y, transform, initializer, ...options} = {}) {
   [x, y] = maybeTuple(x, y);
-  return new DelaunayMark(data, {...options, x, y});
+  // Apply the exclusiveFacets transform before any other transform or initializer.
+  options = basic(options, exclusiveFacets);
+  if (transform != null) options = basic(options, transform);
+  options = {...basic(options, exclusiveFacets), initializer, x, y};
```

Comment:
Is this applying the exclusiveFacets transform twice? It should be harmless but I wonder if and why it is necessary.

Reply:
reworked that part — the solution was indeed much simpler (and, trying to break it, I found another potential crash!)